src/templates/tests/unit/Component.ts

```diff
@@ -0,0 +1,23 @@
+const { registerSuite } = intern.getInterface('object');
+
+import harness, { Harness } from '@dojo/test-extras/harness';
+
+import <%= name %>, { <%= name %>Properties } from './<%= testComponentPath %>';
+
+let widget: Harness<<%= name %>Properties, typeof <%= name %>>;
+
+registerSuite('<%= name %>', {
```

Comment:
Can we switch this to the `bdd` interface? I think we have done that for the `create-app` scaffold too